users/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
 
-from users.views import KakaoSignInView, KakaoCallBackView
+from users.views import KakaoSignInView, KakaoCallBackView, ProfileView
 
 urlpatterns = [
     path('/signin/kakao',KakaoSignInView.as_view()),
     path('/signin/kakao/callback',KakaoCallBackView.as_view()),
+    path('/mypage',ProfileView.as_view()),
```

Comment:
엔드포인트 주소는 page 단위가 아닌 데이터를 기준으로 네이밍 해주세요.
```suggestion
    path('/profile',ProfileView.as_view()),
```